pkg/reconciler/approvaltask/utils.go

```diff
@@ -161,3 +154,204 @@ func (c *Reconciler) updateLabelsAndAnnotations(ctx context.Context, run *v1beta
 	}
 	return nil
 }
+
+func createApprovalTask(ctx context.Context, approvaltaskClientSet versioned.Interface, run *v1beta1.CustomRun) (v1alpha1.ApprovalTask, error) {
+	var (
+		approvals []v1alpha1.Input
+		users     []string
+		err       error
+	)
+
+	logger := logging.FromContext(ctx)
+	approvalsRequired := 1
+
+	for _, v := range run.Spec.Params {
+		var pa v1alpha1.Input
+
+		if v.Name == "approvals" {
```

Comment:
create constants and move `approvals`, `approvalsRequired`, `wait`, and other string references